mcs/class/System/Mono.Net.Security/SystemCertificateValidator.cs

```diff
@@ -45,7 +45,8 @@ static SystemCertificateValidator ()
 #elif MONODROID
 			is_macosx = false;
 #else
-			is_macosx = System.IO.File.Exists (OSX509Certificates.SecurityLibrary);
+			// make it work on wine too.
+			is_macosx = Environment.OSVersion.Platform == PlatformID.MacOSX;
```

Comment:
We return PlatformID.Unix instead of MacOSX on osx for compatibility reasons so this won't work.

Can you explain why the check for that file is not enough on Wine?